### ISSUE TITLE
Better document the package

### ISSUE THREAD

**Issue by kenperkins** [zoomhub/node-deepzoomtools#3]
This library would benefit from some documentation above the usage in `test/_index.coffee`

**Comment by somentelucas** [comment]
agreed

**Comment by soichih** [comment]
I have no idea how to run this.. Can you document?

**Comment by gasi** [comment]
@soichih Does this example help? https://github.com/zoomhub/node-deepzoomtools/blob/331d6daafcdc7ca007f7b95a1dd820121214b0be/test/index._coffee#L39-L50

**Comment by soichih** [comment]
@gasi Yes, I think it will be nice if it's explained on the README.md. Also, can you post it in a plain Javascript?

**Comment by gasi** [comment]
@soichih I am currently not maintaining this library, but feel free to submit a PR and I’d be happy to review and merge. Thanks for your understanding 😄
